project.clj

```diff
@@ -28,27 +30,34 @@
                          [net.logstash.logback/logstash-logback-encoder "4.7"]
                          [org.codehaus.janino/janino "2.7.8"]
                          [com.fasterxml.jackson.core/jackson-core "2.7.3"]
+                         [org.yaml/snakeyaml "1.13"]
```

Comment:
To generalize my previous questions a bit: is the goal to put versions for all dependencies in this file or just the ones that are shared across projects?  Looking at some of the other dependencies added, it's not clear why they would be used in more than one of the projects this PR adds support for ... but perhaps they are transitive dependencies.